coreos-installer

```diff
@@ -378,6 +378,43 @@ write_networking_opts() {
     echo "set ignition_network_kcmdline=\"$(cat /tmp/networking_opts)\"" >> /mnt/boot_partition/ignition.firstboot
 }
 
+#########################################################
+# Helper to persist ignition platform id to initramfs first boot
+#########################################################
+write_platform_id() {
+    local tmp_platform_id="/tmp/platform_id"
+    local proc_cmdline="/proc/cmdline"
+
+    if [ ! -f ${tmp_platform_id} ]; then
+        log "Not overwriting ignition platform id, no platform id provided"
+        return
+    else
+        local platform_id="$(cat ${tmp_platform_id})"
+    fi
+
+
+    log "Overwriting ignition platform id"
+    # check for the boot partition
+    mkdir -p /mnt/boot_partition
+    mount_boot_partition /mnt/boot_partition
+    trap 'umount /mnt/boot_partition; trap - RETURN' RETURN
+    local conf_files=$(ls /mnt/boot_partition/loader/entries/ | grep 'conf')
```

Comment:
what about `ls /mnt/boot_partition/loader/entries/*conf` instead?

Reply:
this will also give us an absolute path and make it so you don't have to do `local conf_files_path="/mnt/boot_partition/loader/entries/$f"` below.

Reply:
Yes, you're right. Thanks for pointing that out!